### ISSUE TITLE
Create a RealWorld example in Dream (and Melange? ReScript? for full-stack experience)

### ISSUE THREAD

**Issue by aantron** [aantron/dream#142]
See https://github.com/gothinkster/realworld.

Thanks to @thangngoc89 for suggesting.

**Comment by outkine** [comment]
I completely agree that a serious demo app would be useful for both beginners and for finding opportunities to improve Dream itself, but I have some doubts about RealWorld specifically. [It's had some problems with finding maintainers](https://github.com/gothinkster/realworld/issues/578), and the new website presentation structure is strange (they've replaced a README that showed a list of projects sorted by stars with a flat list on a custom site...very hard to use).

On the other hand, if there's no other opportunities to make an example app, then we might as well go with RealWorld.
